### PR TITLE
auto open docs on serve

### PR DESCRIPTION
**Pull request by jrappen**
TODO: please add chinese translation

### CODE REVIEW

File: bin/docsify [hunk 1]
```diff
@@ -45,10 +45,18 @@ var argv = require('yargs')
     handler: (argv) => run.init(argv.path, argv.local, argv.theme)
   })
   .command({
-    command: 'serve <path> [--port 3000]',
+    command: 'serve <path> [--disable-auto-open false] [--port 3000]',
```
Comment (QingWei-Li) [inline bin/docsify]: I think we should use `--open`.

- `disable-auto-open` is too long
- The default auto open browser is not friendly

Comment (jrappen) [inline bin/docsify]: Ok.

**Comment by jrappen** [comment]
@QingWei-Li The chinese translation is still missing, as I don't speak it.

**Comment by QingWei-Li** [comment]
@jrappen Done.

**Pull request merged**